packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -1,141 +1,168 @@
+"use strict"
```

Comment:
Need to revert to source

Reply:
Should be reverted with latest commit. Sorry!